src/main/java/tokio/commands/AddCommand.java

```diff
@@ -0,0 +1,70 @@
+package tokio.commands;
+
+import java.io.IOException;
+
+import tokio.exceptions.DukeException;
+import tokio.storage.Storage;
+import tokio.tasks.Deadlines;
+import tokio.tasks.Events;
+import tokio.tasks.TaskList;
+import tokio.tasks.Todos;
+import tokio.ui.Ui;
+
+/**
+ * Adds todo, deadline and event command to tasklist and storage.
+ */
+public class AddCommand extends Command {
+    protected String cmdLine;
+
+    public AddCommand(String cmdLine) {
+        this.cmdLine = cmdLine;
+    }
+
+    @Override
+    public String execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
Maybe you can make this function easier to be understood by providing javadocs, though the name might be self-explanatory to some extent